pkg/monitoring/doc.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package monitoring
+
+/*
+Package moinitoring manages resources used by Filebeats, Journalbeats, and NodeExporters.  Filebeats and Journalbeats ship
```

Comment:
Is this a new file?  Should it have some logic in it?

Typo "moinitoring"

Reply:
This is a package doc file.  For example: https://golang.org/src/encoding/gob/doc.go
Will fix spelling